tests/compilation_tests/traverse_standard_test.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+import math
+import unittest
+
+from engine.database import DataBase
+from engine.grid import *
+from engine.robot import Robot
+from collections import deque
+
+target = [0, 0]
+add_to_x = False
+gps_noise_range = .3
+
+
+# Takes approx. 9 seconds for 5 tests
+class TestTraverseStandardFunctions(unittest.TestCase):
+
+    def test_init(self):
+        r2d2 = Robot(0, 0, math.pi / 2, epsilon=0.2, max_v=0.5, radius=0.2, init_phase=2)
+        database = DataBase(r2d2)
+
+        allowed_dist_error = 0.5
+        grid = Grid(42.444250, 42.444599, -76.483682, -76.483276)
+        grid_mode = "lawn_border"
+        all_waypoints = grid.get_waypoints(grid_mode)
+        waypoints_to_visit = deque(all_waypoints)
+        unvisited_waypoints = r2d2.traverse_standard(waypoints_to_visit, allowed_dist_error, database)
+        self.assertEqual(deque([]), unvisited_waypoints)
+
+    def test_different_init_robot_pos(self):
+        r2d2 = Robot(42, -76, math.pi / 2, epsilon=0.2, max_v=0.5, radius=0.2, init_phase=2)
+        database = DataBase(r2d2)
+
+        allowed_dist_error = 0.5
+        grid = Grid(42.444250, 42.444599, -76.483682, -76.483276)
+        grid_mode = "lawn_border"
+        all_waypoints = grid.get_waypoints(grid_mode)
+        waypoints_to_visit = deque(all_waypoints)
+        unvisited_waypoints = r2d2.traverse_standard(waypoints_to_visit, allowed_dist_error, database)
+        self.assertEqual(deque([]), unvisited_waypoints)
+
+    def test_different_allowed_dist(self):
```

Comment:
what's the difference between this and test_different_init_robot_pos? It looks like the same code.